ynput/ayon-unreal
# Extract Layout published for other dccs

## Changelog Description
This PR is to make adjustment on the layout product type so that it can publish the layout for not only blender but other DCCs such as Maya.
Prepare for https://github.com/ynput/ayon-maya/issues/65

- [x] Validator for checking the actors for extracting layout
- [x] Version Id supports in Extract Layout
- [x] Temp option for exporting to blender
- [x] Improve the actor selections for layout publishing

## Additional info
The reason to use `instance.data["creator_attributes"]` to store the `members` mainly because the `set_metadata_tag` in imprint function turns any value type to string. Storing the list type in the `creator_attributes` ensure the conversion can be accurately performed. 
Not sure if we want to use `transientData` to store the `members`(I think this is the better solution comparing to `instance.data["creator_attributes"]`

## Testing notes:
1. Create Layout with actors
2. Publish
3. Go to Publish Folder to find the published layout to check if it stores any data
